vulnerabilities/tests/test_api.py

```diff
@@ -23,129 +23,103 @@
 
 import json
 import os
-
-from django.test import TestCase
+import pytest
 
 from vulnerabilities.api import PackageSerializer
 from vulnerabilities.data_dump import debian_dump
 from vulnerabilities.data_dump import ubuntu_dump
 from vulnerabilities.models import Package
 from vulnerabilities.scraper import debian
 from vulnerabilities.scraper import ubuntu
+from django.test import Client
```

Comment:
Would you mind moving this import between the ones from the stdlib and the ones from `vulnerablecode`?